packages/caravan-psbt/src/psbtv2/psbtv2.ts

```diff
@@ -875,7 +942,14 @@ export class PsbtV2 extends PsbtV2Maps {
     map.set(KeyType.PSBT_IN_PREVIOUS_TXID, bw.render());
     bw.writeI32(outputIndex);
     map.set(KeyType.PSBT_IN_OUTPUT_INDEX, bw.render());
+
     if (sequence) {
+      /* If RBF is requested, set the sequence to the RBF opt-in value
+      regardless of the provided sequence
```

Comment:
I think this illustrates my comment above. You would not want to override the sequence. Any value less than `0xfffffffe` would already signal rbf.